/home/ubuntu/.local/lib/python3.10/site-packages/distributed/node.py:183: UserWarning: Port 8787 is already in use.
Perhaps you already have a cluster running?
Hosting the HTTP server on port 38691 instead
  warnings.warn(
Distributing data time: 4.729559898376465
submit_one_func 0.0593717098236084
submit_one_func 0.0006604194641113281
submit_one_func 0.0005555152893066406
submit_one_func 0.0005342960357666016
submit_one_func 0.0005464553833007812
submit_one_func 0.0005249977111816406
submit_one_func 0.0005421638488769531
submit_one_func 0.0005283355712890625
Execution time: 1.1347675323486328
Get table content time: 0.17908978462219238
Get table content time: 0.8296711444854736
Get table content time: 0.6843287944793701
Get table content time: 0.8616001605987549
Get table content time: 0.8410592079162598
Get table content time: 0.8572666645050049
Get table content time: 0.6539428234100342
Get table content time: 0.9723093509674072
Get table content time: 0.8904328346252441
